molsketch/net.sourceforge.molsketch.appdata.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2017 Hendrik Vennekate <mail_AT_domain.tld> -->
+<component type="desktop-application">
+  <id>net.sourceforge.molsketch</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <name>Molsketch</name>
+  <summary>2D Molecular Structures Editor</summary>
+
+  <description>
+    <p>
+    Molsketch is a program for drawing two-dimensional schematic representations of molecules. The goal of the program is to make this quick and easy, without sacrificing the resulting quality. Of course your creations can be used afterwards as vector or bitmap image in any document, presentation or website. Some of the features of Molsketch are:
+    </p>
+    <ul>
+	    <li>Creating regular and consistent bonds easily with tools as a dynamic grid and magnetic atoms</li>
+	    <li>Open, save and import in all formats supported by the OpenBabel™ library</li>
+	    <li>Export to Scalable Vector Graphics (SVG) and a number of common used bitmap formats</li>
+	    <li>Print and export your document to PDF</li>
+	    <li>Intelligent addition/removal of hydrogen atoms</li>
+	    <li>Real-time information about the molecule, like charge and weight</li>
```

Comment:
Ooops, actually gotta fix that functionality...

Reply:
I just copied stuff from the website, you can also remove it and no one will know. I won't tell anyone, I promise!